<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Document Processor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="loadArchiveButton">
      <property name="text">
       <string>Загрузить архив</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="loadFilesButton">
      <property name="text">
       <string>Загрузить файлы</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="clearBufferButton">
      <property name="text">
       <string>Очистить буфер</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="runVerificationButton">
      <property name="text">
       <string>Выполнить сверку</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="viewLogsButton">
      <property name="text">
       <string>Просмотреть логи</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
